src/accountant.rs

```diff
@@ -92,11 +94,11 @@ impl Accountant {
     }
 
     pub fn process_transaction(self: &mut Self, tr: Transaction) -> Result<()> {
-        if !tr.verify() {
+        if !tr.plan_verify() {
```

Comment:
Method names need to be either `<verb>` or `<verb>_<subject>`. Can you change this to `verify_plan()`?